provider/openstack/config.go

```diff
@@ -69,8 +70,13 @@ func (c *environConfig) useDefaultSecurityGroup() bool {
 	return c.attrs[UseDefaultSecgroupKey].(bool)
 }
 
-func (c *environConfig) network() string {
-	return c.attrs[NetworkKey].(string)
+func (c *environConfig) networks() []string {
+	raw := strings.Split(c.attrs[NetworkKey].(string), ",")
```

Comment:
Just to put it here, I thought we might prefer to do this as a YAML list rather than a comma separated string, though I guess we can't really do that for model config anyway.

Reply:
Yes, I had trouble working with this as a YAML list at the command line. 

I went this way because there is precedent in articles like `egress-subnets`.